docs/examples/README.md

```diff
@@ -1,17 +1,14 @@
 # Examples
 
-This directory contains examples of DALI usage. List of examples:
+This directory contains examples of DALI usage.
```

Comment:
We should make this readme appear in the proper docs instead of having it only on GH. But it can be done as a followup. I would also list what the `General` means.